### ISSUE TITLE
switch indentation

### ISSUE THREAD

**Issue by jandamm** [nicklockwood/SwiftFormat#152]
I know that Xcode itself indents (control + I) like this:

```
switch type {
case .a: break
case .b:
    break
}
```
Which matches with Swiftformats indetation.

Apples Demo Projects on the other hand are indented like this:

```
switch type {
    case .a: break
    case .b:
        break
}
```

In my opinion this is more readable and feels more natural.

I wanted to ask if you could integrate this as an option.
Thanks in advance!

**Comment by nicklockwood** [comment]
Yeah, there have been a few requests for this. I'll try to add it soon.

**Comment by nicklockwood** [comment]
Implemented in 0.30.0 - sorry it took so song!
